Makefile

```diff
@@ -215,15 +215,15 @@ cargo-update:
 
 .PHONY: build-wasm-mandala
 build-wasm-mandala:
-	./scripts/build-only-wasm.sh --profile production -p mandala-runtime --features=on-chain-release-build --workspace --exclude runtime-integration-tests --exclude e2e-tests
+	./scripts/build-only-wasm.sh --profile production -p mandala-runtime --features=on-chain-release-build --package mandala-runtime
```

Comment:
why this change? there is already `-p mandala-runtime`

Reply:
Because all we want is the mandala-runtime, so it has to compile one package and not entire workspace

Reply:
Old command will build relaychain runtime as well

Reply:
Oh I think I missed it

Reply:
Will fix tomorrow